src/store/Modules/Queries.ts

```diff
@@ -77,10 +77,11 @@ export default class Queries extends VuexModule {
       this.context.commit("addGlobalQueries", data)
     } catch (err) {
       console.log(err)
+      this.context.commit("addGlobalQueries", {})
```

Comment:
Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.